checkpoint/keeper.go

```diff
@@ -185,10 +185,10 @@ func (k *Keeper) GetLastCheckpoint(ctx sdk.Context) (hmTypes.Checkpoint, error)
 	return _checkpoint, cmn.ErrNoCheckpointFound(k.Codespace())
 }
 
-// GetHeaderKey appends prefix to headerNumber
-func GetHeaderKey(headerNumber uint64) []byte {
-	headerNumberBytes := []byte(strconv.FormatUint(headerNumber, 10))
-	return append(HeaderBlockKey, headerNumberBytes...)
+// GetHeaderKey appends prefix to checkpointNumber
+func GetHeaderKey(checkpointNumber uint64) []byte {
```

Comment:
```suggestion
func GetHeaderKey(checkpointNumber uint64) []byte {
```
func GetCheckpointKey(checkpointNumber uint64) []byte {